tests/test_api.py

```diff
@@ -265,7 +265,8 @@ def test_geojson(self):
         geojson = filt.GeoJson(shape)
         assert geojson["type"] == "Polygon"
         assert isinstance(geojson["coordinates"], list)
-        assert isinstance(geojson["coordinates"][0], filt.Coordinate)
+        assert len(geojson["coordinates"][0]) >= 4  # a polygon need to have more than  4 points
```

Comment:
Well, 3 points would be enough for a polygon (triangle)... But I understand that it is rare to define a region of interest with a triangle!